drivers/hmis/app/graphql/types/hmis_schema/referral_posting_input.rb

```diff
@@ -0,0 +1,18 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class HmisSchema::ReferralPostingInput < BaseInputObject
+    argument :status_note, String, required: false
+    argument :status, ID, required: false
+    argument :denial_reason, ID, required: false
+    argument :denial_note, String, required: false
```

Comment:
@ttoomey denial note will be part of the [denial screen update](https://www.pivotaltracker.com/story/show/185282623) along with the "re request" option and a "referral result." Seems fine to use the same mutation for that, there will just be some conditional validations based on the status.